r2/r2/lib/template_helpers.py

```diff
@@ -607,11 +607,11 @@ def simplified_timesince(date, include_tense=True):
     if date > timeago("1 minute"):
         return _("just now")
 
-    since = []
-    since.append(timesince(date))
+    since = timesince(date)
     if include_tense:
-        since.append(_("ago"))
-    return " ".join(since)
+        return _("%s ago") % since
```

Comment:
I think if you do this like `_("%(time)s ago")` it will be clearer for the translators.